doc/source/contribution/documentation/introduction.rst

```diff
@@ -13,10 +13,11 @@ Groonga uses Sphinx_ as documentation tool.
 
 .. _Sphinx: https://www.sphinx-doc.org/
 
-Here are command lines to install Sphinx.
+Here are command lines to install Sphinx and the other dependencies.
```

Comment:
We are describing about Sphinx in here.
So, in my opinion, we should split description about Sphinx and `setup.sh`.

Because Sphinx have less to do with `setup.sh`.

Reply:
It sounds nice to me. I will fix it like the following!
- Also, `How to Install Sphinx` can be referred by i18n page too.
```
## How to Install depended packages
## How to Install Sphinx
```

Reply:
fix: [8ded51e](https://github.com/groonga/groonga/pull/1715/commits/8ded51e3bc3d5a5e93a16f4c81aa25c4210a594f)
- I added the section for focusing on installing Sphinx.
fix: [c7be83c](https://github.com/groonga/groonga/pull/1715/commits/c7be83c6dd2d38817693cbb3891c9532289a8a4d)
- I add the explanation about why we need install required packages.